node/node.go

```diff
@@ -48,6 +50,7 @@ var (
 type Node struct {
 	datadir  string         // Path to the currently used data directory
 	eventmux *event.TypeMux // Event multiplexer used between the services of a stack
+	accman   *accounts.Manager
```

Comment:
What should the comment say? I seriously don't know what to write about this field. It's kind of self-explanatory that this is the account manager.